packagers/jam.go

```diff
@@ -32,9 +33,21 @@ func (j Jam) WithExecutable(executable Executable) Jam {
 }
 
 func (j Jam) Execute(buildpackDir, output, version string, offline bool) error {
+
+	extensionTomlPath := filepath.Join(buildpackDir, "extension.toml")
+
+	buildpackOrExtensionToml := "buildpack.toml"
+	command := "--buildpack"
+	if _, err := os.Stat(extensionTomlPath); err == nil {
+		buildpackOrExtensionToml = "extension.toml"
+		command = "--extension"
+	}
+
+	fmt.Println("jam", command, filepath.Join(buildpackDir, buildpackOrExtensionToml), "--output", output, "--version", version, "--offline", offline)
```

Comment:
I'd probably remove this println unless you have a compelling reason to keep it in. I'd prefer to err on the side of silence - especially in the happy-path. I don't think we have any good existing model for debug logging in occam, and if/when we create that, I'd like to apply it universally.

Reply:
> In principle I think this is fine, though I have a couple of minor changes I'd like to see. Once we've updated them I'd be happy to merge.
> 
> In general, I don't love the idea of silently selecting buildpack.toml vs extension.toml or silently falling back from one to the other, but I'm having a hard time coming up with a better interface at this time. So I'm happy with this general direction.

I also dont agree with the idea of fallback, but for now looks like a good alternative as I tried adding an extra argument on the `func (j Jam) Execute(buildpackDir, output, version string, offline bool) error {` and it turned out with a lots of errors.

Added your suggestions, I guess next step before merging this pr is to wait for the https://github.com/paketo-buildpacks/jam/pull/234